Automattic/wp-calypso
# Subscriptions Management: Add comments filtering

## Proposed Changes

This PR adds filter to Comments tab.

## Testing Instructions

* Run this PR on your local env
* Go to http://calypso.localhost:3000/subscriptions/comments
* Verify if the filter is working as expected for P2 filtering option (to filter P2s you can use your A8C account)

<img width="748" alt="Screenshot 2023-04-19 at 17 48 49" src="https://user-images.githubusercontent.com/3113712/233205459-74e2690a-a31f-4ad6-b6e1-127deddb9174.png">

## Pre-merge Checklist
